database/models/core.py

```diff
@@ -55,14 +66,10 @@ class Account(CodecovBaseModel, MixinBaseClass):
     plan_auto_activate = Column(types.Boolean, nullable=False, default=True)
     is_delinquent = Column(types.Boolean, nullable=False, default=False)
 
-    users = relationship("User", secondary="accounts_users", backref="accounts")
-
-
-class AccountsUsers(CodecovBaseModel):
-    __tablename__ = "accounts_users"
-    account_id = Column(types.BigInteger, ForeignKey("accounts.id"), primary_key=True)
-    user_id = Column(types.BigInteger, ForeignKey("users.id"), primary_key=True)
-    created_at = Column(types.DateTime, nullable=False, default=datetime.utcnow)
+    users = relationship(
+        "User", secondary="codecov_auth_accountsusers", back_populates="accounts"
+    )
+    organizations = relationship("Owner", back_populates="account")
```

Comment:
We don't have orgs in account do we?

Reply:
I think this is how you represent a many-to-many in sqlalchemy, from [these docs](https://docs.sqlalchemy.org/en/20/orm/basic_relationships.html#setting-bi-directional-many-to-many).  You have to explicitly set the `related_name` field, so it looks different from django but is the same thing